requirements_for_test.txt

```diff
@@ -0,0 +1 @@
+pytest==2.8.2
```

Comment:
Minor, I think the file should be named `test_requirements.txt`, otherwise tab complete stops before `.txt` when installing the application dependencies from `requirements.txt`